--- ./misc/paths.py.in.orig	2019-04-22 15:19:19.000000000 +0900
+++ ./misc/paths.py.in	2019-04-22 17:34:25.379532732 +0900
@@ -61,11 +61,17 @@ sys.path.insert(0, os.path.join(prefix,
 sitedir = os.path.join(sys.prefix, 'lib', 'python'+sys.version[:3],
                        'site-packages')
 sys.path.append(sitedir)
+sitedir64 = os.path.join(sys.prefix, 'lib64', 'python'+sys.version[:3],
+                         'site-packages')
+sys.path.append(sitedir)
 
 # Include Python's dist-packages directory.
 distdir = os.path.join(sys.prefix, 'lib', 'python'+sys.version[:3],
                        'dist-packages')
 sys.path.append(distdir)
+distdir64 = os.path.join(sys.prefix, 'lib64', 'python'+sys.version[:3],
+                         'dist-packages')
+sys.path.append(distdir)
 
 
 # In a normal interactive Python environment, the japanese.pth and korean.pth
