diff --git a/contrib/courier-to-mailman.py b/contrib/courier-to-mailman.py
index 922401f..572b47d 100644
--- a/contrib/courier-to-mailman.py
+++ b/contrib/courier-to-mailman.py
@@ -1,4 +1,4 @@
-#! /usr/bin/python
+#! /usr/bin/python2
 
 # Configuration variables - Change these for your site if necessary.
 #
diff --git a/tests/onebounce.py b/tests/onebounce.py
index 85f3b98..b54d075 100755
--- a/tests/onebounce.py
+++ b/tests/onebounce.py
@@ -1,4 +1,4 @@
-#! /usr/bin/python
+#! /usr/bin/python2
 
 # Copyright (C) 2002-2018 by the Free Software Foundation, Inc.
 #
diff --git a/bin/msgfmt.py b/bin/msgfmt.py
index 8a2d4e6..27a40bf 100755
--- a/bin/msgfmt.py
+++ b/bin/msgfmt.py
@@ -1,4 +1,4 @@
-#! /usr/bin/python
+#! /usr/bin/python2
 # -*- coding: iso-8859-1 -*-
 # Written by Martin v. Löwis <loewis@informatik.hu-berlin.de>
 
diff --git a/Mailman/Archiver/pipermail.py b/Mailman/Archiver/pipermail.py
index cfca480..3d6a9d6 100644
--- a/Mailman/Archiver/pipermail.py
+++ b/Mailman/Archiver/pipermail.py
@@ -1,4 +1,4 @@
-#! /usr/bin/python
+#! /usr/bin/python2
 
 from __future__ import nested_scopes
 
diff --git a/Mailman/Archiver/pipermail.py.unicode b/Mailman/Archiver/pipermail.py.unicode
index 62fb92e..a663829 100644
--- a/Mailman/Archiver/pipermail.py.unicode
+++ b/Mailman/Archiver/pipermail.py.unicode
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python2
 
 from __future__ import nested_scopes
 
diff --git a/Mailman/Post.py b/Mailman/Post.py
index 7f86696..b4dcfb0 100644
--- a/Mailman/Post.py
+++ b/Mailman/Post.py
@@ -1,4 +1,4 @@
-#! /usr/bin/python
+#! /usr/bin/python2
 #
 # Copyright (C) 2001-2018 by the Free Software Foundation, Inc.
 #
diff --git a/tests/fblast.py b/tests/fblast.py
index 092e420..905401d 100644
--- a/tests/fblast.py
+++ b/tests/fblast.py
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python2
 """Throw email at Mailman as fast as you can.
 
 This is not a unit test, it's a functional test, so you can't run it within
